guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -495,6 +495,12 @@ You can invoke `test_jdbcmysql`, `test_jdbcsqlite3` or `test_jdbcpostgresql` als
 
 To use an external debugger (pry, byebug, etc), install the debugger and use it as normal.  If debugger issues occur, run tests in serial by setting `PARALLEL_WORKERS=1` or run a single test with `-n test_long_test_name`.
 
+If running tests against generators you will need to set `PRINT_STDOUT=true` in order for debugging tools to work.
+
+```sh
+PRINT_STDOUT=true ./bin/test test/generators/actions_test.rb
+```
```

Comment:
I wonder if we should add this to [Creating and Customizing Rails Generators & Templates](https://edgeguides.rubyonrails.org/generators.html)

Reply:
Yes, we should. The one you change is only internal, but this variable also changes public interface of generator testing.

Reply:
Updated in 7ff16d940818441acbb4c6ce3816245579e5a2ea.